docs/MAINTAINERS-guidelines.md

```diff
@@ -0,0 +1,57 @@
+# Introduction
+All Hyperledger projects MUST have a `MAINTAINERS` file (`MAINTAINERS.md` or `MAINTAINERS.rst`) at the top-level directory of the source code. This document will provide guidelines on what should be included in the `MAINTAINERS` file.
+
+# Guidelines
+## List of Project Maintainers
+The first thing that MUST be included in the `MAINTAINERS` file is a list of the project's maintainers, both active and emeritus.
+
+It is recommended that the lists be sorted alphabetically and contain the maintainers name, GitHub ID, LFID, Chat ID, Email, Company Affiliation, and Scope.
```

Comment:
Is it clear that we recommend at least those fields and that it is not forbidden to add other fields?

Reply:
This is only a recommended list of fields. Do you have a suggestion on re-wording?